indexing-service/src/main/java/org/apache/druid/indexing/overlord/TaskLockbox.java

```diff
@@ -1383,6 +1383,20 @@ Map<String, NavigableMap<DateTime, SortedMap<Interval, List<TaskLockPosse>>>> ge
     return running;
   }
 
+  Set<TaskLock> getLocksForDatasource(final String datasource)
```

Comment:
I guess this is not used anymore since we have already added the `/activeLocks` API.

Reply:
Removed